x-pack/legacy/plugins/actions/server/plugin.ts

```diff
@@ -74,15 +74,15 @@ export class Plugin {
             all: ['action', 'action_task_params'],
             read: [],
           },
-          ui: [],
+          ui: ['save', 'show', 'delete'],
```

Comment:
It would make sense but when I started to look at the other places, they use `show` terminology (including docs) so I figured keeping it for consistency.

Reply:
ok makes sense to me to keep it consistent